### ISSUE TITLE
Login + account creation pages should be served over https. 

### ISSUE THREAD

**Issue by gabeguz** [gophergala2016/GoBlog#2]
https://www.goblog.pw/signup/

"This webpage is not available

ERR_CONNECTION_REFUSED"

**Comment by AnneTheAgile** [comment]
ty @gabeguz , I agree signup should be on https. I think Let's Encrypt is almost ready, right? Then @farazfazli and we would be able to add support. 

Did you get to create a blog for yourself ? 

We're continuing at https://github.com/GetGoBlog  , so I'll go reference this ticket there.

**Comment by gabeguz** [comment]
Yes, letsencrypt is ready... at least, I'm already using it for some projects so I believe it's available.   I was able to create a blog, yes: http://gabe.goblog.pw
